File: Herencia/ejercicio 5/Python/Desarrollador.py
from Empleado import Empleado
class Desarrollador(Empleado):
    def __init__(self, nombre, apellido, salarioBase, añosDeAntiguedad, lenguaje_programacion, horas_extras):
        super().__init__(nombre, apellido, salarioBase, añosDeAntiguedad)
        self.lenguaje_programacion = lenguaje_programacion
        self.horas_extras = horas_extras
    def calcular(self):
        extra = self.horas_extras * 20
        return super().calcular() + extra

File: Herencia/ejercicio 5/Java/Desarrollador.java
class Desarrollador extends Empleado {
    public String lenguaje;
    public int horas;
    public Desarrollador(String nombre, String apellido, double salario, int añosAnti, String lenguaje, int horas) {
        super(nombre, apellido, salario, añosAnti);
        this.lenguaje = lenguaje;
        this.horas = horas;
    }
    @Override
    public double calcularSalario() {
        return super.calcularSalario() + (horas * 20);
    }
    public int getHorasExtras() {
        return horas;
    }
}